src/components/app.vue

```diff
@@ -94,14 +94,21 @@
         }
 
         // Only use main level projects API if tighter scope not defined
-        const scope = Config.root.projectScope
-        const scopeId = Config.root.projectScopeId
-        let urlPrefix = ''
-        if ((scope === 'users' || scope === 'groups') && scopeId !== null) {
-          urlPrefix = '/' + scope + '/' + scopeId
+        const scopeType = Config.root.projectScope
+        // Reformat the variable as a flat list of Ids
+        const scopeId = [Config.root.projectScopeId].flat()
+
+        var apiPromises = []
```

Comment:
`var` :)

Reply:
according to the workflow I have modified, I don't see any other way to do it without `var`.

Reply:
You should be able to use `const` here. You don't assign any new value to it and you use `apiPromises` only in same-level or below scoped blocks.

Reply:
Does it throw an error if you use `const`?

Reply:
Well... I didn't know this was possible. Thank's for the information, I just made the modification